deploy/helm-chart/values.yaml

```diff
@@ -58,6 +58,9 @@ nginx:
     tag: edge
     pullPolicy: Always
 
+  ## The dynamic configuration for the data plane that is contained in the NginxProxy resource.
```

Comment:
I suggest removing dynamic configuration, because dynamic NGINX configuration historically and is still used to describe different configuration.